rll-examples/disable-monitoring.sh

```diff
@@ -69,6 +69,15 @@ if [[ -e $collectd_conf ]]; then
     fi
   done
 
+  # Older (v13) style -- all-in-one config
+  if $sudo grep -E 'Server.*rightscale.com' "$collectd_conf" 2>/dev/null; then
+    echo "RightScale collectd-based monitoring is enabled. Disabling collectd configuration in $collectd_conf."
+    backup_time=$(date -u +%Y%m%d%H%M%S)
+    $sudo cp "${collectd_conf}" "${collectd_conf}.${backup_time}"
+    perl -0777 -pi -e 's/LoadPlugin network.*?rightscale.*?Plugin>//is' "${collectd_conf}"
```

Comment:
If we require `sudo` in the `cp` command above, will editing the file with perl not require `sudo`?